subprojects/docs/src/docs/userguide/running-builds/build_environment.adoc

```diff
@@ -32,15 +32,16 @@ Aside from configuring the build environment, you can configure a given project
 [[sec:gradle_configuration_properties]]
 == Gradle properties
 
-Gradle provides several options that make it easy to configure the Java process that will be used to execute your build. While it's possible to configure these in your local environment via `GRADLE_OPTS` or `JAVA_OPTS`, it is useful to store certain settings like JVM memory configuration and Java home location in version control so that an entire team can work with a consistent environment.
+Gradle provides several options that make it easy to configure the Java process that will be used to execute your build. While it's possible to configure these in your local environment via `GRADLE_OPTS` or `JAVA_OPTS`, it is useful to be able to store certain settings like JVM memory configuration and Java home location in version control so that an entire team can work with a consistent environment. To do so, place these settings into a `gradle.properties` file committed to your version control system.
 
-Setting up a consistent environment for your build is as simple as placing these settings into a `gradle.properties` file. The configuration is a combination of all your `gradle.properties` files, but if an option is configured in multiple locations, the _first one wins_:
+The final configuration taken into account by Gradle is a combination of all your `gradle.properties` files. If an option is configured in multiple locations, the _first one_ found in any of these locations wins:
 
-* system properties, e.g. when `-Dgradle.user.home` is set on the command line.
```

Comment:
Shouldn't be removed, system properties on the command line always win. This is not just about `-Dgradle.user.home` (which is a bad example), but any system prop you wanna pass to the build.

Reply:
To me, it was confusing why system properties are mentioned in this section to start with. This section is about *Gradle* properties after all. I know that you can also set system properties inside `gradle.properties`, so in a way, system properties can be seen as a subset of Gradle properties. And it does not make sense to me to limit the docs to a subset of Gradle properties here.

Reply:
You can mention `-P` then. I just want it to be clear that the command line wins over all the files.

Reply:
Done.